assets/src/edit-story/elements/text/edit.js

```diff
@@ -48,13 +48,18 @@ import calcRotatedResizeOffset from '../../utils/calcRotatedResizeOffset';
 import generatePatternStyles from '../../utils/generatePatternStyles';
 import useRichText from '../../components/richText/useRichText';
 import { useTransformHandler } from '../../components/transform';
-import { generateParagraphTextStyle, getHighlightLineheight } from './util';
+import {
+  calcFontMetrics,
+  generateParagraphTextStyle,
+  getHighlightLineheight,
+} from './util';
 
 // Wrapper bounds the text editor within the element bounds. The resize
 // logic updates the height of this element to show the new height based
 // on the content and properties.
 const Wrapper = styled.div`
   ${elementFillContent}
+  ${({ backgroundColor }) => backgroundColor && elementWithBackgroundColor}
```

Comment:
Maybe this test for unset value could be moved to the [`elementWithBackgroundColor` definition](https://github.com/google/web-stories-wp/blob/8d0153c9f86507fb463be4c74d11f5c59519416a/assets/src/edit-story/elements/shared/index.js#L54) instead?

Alternatively, if `generatePatternStyles` receives `null`, it [returns `transparent` for the property](https://github.com/google/web-stories-wp/blob/8d0153c9f86507fb463be4c74d11f5c59519416a/assets/src/edit-story/utils/generatePatternStyles.js#L104-L106). That check might just be extended to any _falsey_ value, though I don't know what other values could appear here (`undefined` will be default'ed to `null`)?

Reply:
Moved to elementWithBackgroundColor definition.